lib/xlsx/xform/book/workbook-xform.js

```diff
@@ -157,7 +157,7 @@ class WorkbookXform extends BaseXform {
       if (!rel) {
         return;
       }
-      worksheet = model.worksheetHash[`xl/${rel.Target}`];
+      worksheet = model.worksheetHash[`xl/${rel.Target.replace(/^(\s|\/xl\/)+/, '')}`];
```

Comment:
could you please add a comment here as to what strings would be transformed here? I.e., anything with a preceding `/xl/`?

Reply:
@alubbe yes, if `rel.Target` start with `[space]/xl/` or `/xl/`  , then it will be replaced with `''` and spliced behind  `xl/`,  otherwise it will be spliced directly behind `xl/`.  i.g.
```
'/for/bar' ---> 'xl/for/bar'
'   /xl/for/bar' ---> 'xl/for/bar'
'/xl/for/bar' ---> 'xl/for/bar'
```